airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -47,6 +47,7 @@ public class EnvConfigs implements Configs {
   public static final String AIRBYTE_API_AUTH_HEADER_VALUE = "AIRBYTE_API_AUTH_HEADER_VALUE";
   public static final String WORKER_ENVIRONMENT = "WORKER_ENVIRONMENT";
   public static final String SPEC_CACHE_BUCKET = "SPEC_CACHE_BUCKET";
+  public static final String LOCAL_CATALOG_PATH = "LOCAL_CATALOG_PATH";
```

Comment:
nit: to make consistent with `REMOTE_CONNECTOR_CATALOG_URL` and not overload "catalog" too much

```suggestion
  public static final String LOCAL_CONNECTOR_CATALOG_PATH = "LOCAL_CONNECTOR_CATALOG_PATH";
```